app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -95,7 +95,11 @@ $include-html-global-classes: $include-html-classes;
 // $font-weight-normal: normal;
 // $font-weight-bold: bold;
 
-// $white       : #FFFFFF;
+$white       : #FFFFFF;
+$kellygreen : #42A88F;
```

Comment:
Color `#42A88F` should be written as `#42a88f`<br>Variable names should be followed immediately by a colon